index.js

```diff
@@ -1,153 +1,157 @@
-"use strict"
+'use strict';
 
 module.exports = {
-    "env": {
-        "browser": true,
-        "node": false,
-        "es6": true
+    env: {
+        browser: true,
+        node: true,
+        es2022: true,
+        amd: true,
     },
-    "globals": {
-        "require": true,
-        "module": true
+    extends: 'eslint:recommended',
+    plugins: ['import'],
```

Comment:
No prettier? 😉

Reply:
@mscharl Not for this version, but I'm sure we'll implement it in the near future.